lib/logflare/google/resource_manager.ex

```diff
@@ -123,6 +110,11 @@ defmodule Logflare.Google.CloudResourceManager do
         members: ["serviceAccount:#{env_service_account()}"],
         role: "roles/resourcemanager.projectIamAdmin"
       },
+      %Model.Binding{
+        condition: nil,
+        members: ["serviceAccount:#{env_cloud_build_sa()}"],
+        role: "roles/cloudbuild.builds.editor"
+      },
```

Comment:
I think this is the relevant change 😅 but had to clean up a bit for readability sake